update signup (#44)
tidbcloud/dbaas-docs: tidb-cloud-quickstart.md

Now you can visit TiDB Cloud to apply for a trial.

> 1. Visit <https://tidbcloud.com/signup>, and submit the following information:

Reviewer: Please confirm whether this is the right link because it seems that it can't be visited successfully.

Author: yes, because it is still in dev now, the link will be valid when we launch it.

Reviewer: @h293wang-new Got it. Then for such cases, you can provide the info in the PR **Conversation** box and add a **status/DNM** label.

    - First name
    - Last name
    - Work email
    - Job function
    - Company name
    - Phone number
    - Password

> 2. Verify your email address 
> Please check your Spam/Junk email folder in case they are being delivered there instead of your inbox. Click the verification link in the email to access your TiDB Cloud account.

Reviewer: - Generally, avoid using the word "please" in ordinary user documentation, which is over-polite.
- Leave a blank line before a paragraph or after a list.
- End a sentence with a period.
```suggestion
2. Verify your email address.

    If you cannot find the verification email in your inbox, check your Spam or Junk email folder. Click the verification link in the email to access your TiDB Cloud account.
```

> 3. Choose your trial option 

Reviewer: - Do not leave an extra space at the end of a line. We have strict format requirements to keep our documentation clean and clear.
```suggestion
3. Select your trial plan.
```

> Choose your trial option from these three following plans: 48-hour free trial, Proof of Concept, and Discounted On-demand Trial. Or you can click <dismiss> to choose later.

Reviewer: - Please confirm the **capitalization** of the three plans, and the "dismiss" (are all letters in lower case?). Do keep them consistent with that in the UI.
```suggestion

    Select your trial plan from the following three ones or click **dismiss** to select one later.

    - 48-hour free trial
    - Proof of Concept
    - Discounted On-demand Trial
```

Author: -48 hour Free Trial
-2 Week POC Free Trial
-Preview On-Demand

Reviewer: > -48 hour Free Trial
> -2 Week POC Free Trial
> -Preview On-Demand

@h293wang-new Just to confirm, are these the exact words displayed? And would you please tell the capitalization of "dismiss"?

Author: On TiDB sign up page:
-48 Hour Free Trial
-2 Week POC Free Trial
-Preview On-Demand

On TiDB Console offer selection page:
-Free Trial
-Proof of Concept
-Preview On-Demand
-Dismiss

Do you prefer to use the content exactly the same as the console's offer selection page?
